### PR TITLE
update docs to remove cookiecutter and add asset bundle templates

### PR DESCRIPTION
**Pull request by mingyu89**
As title. 
Update docs.

migration plan https://docs.google.com/document/d/19MOK9o-f0A_-NMmxC1cJGQSQ2WkJXZ4vXQZ4plu-tng/edit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,40 +35,39 @@ structure defined in the default stack.
...
 
 This will prompt for parameters for project initialization. Some of these parameters are required to get started:
- * ``project_name``: name of the current project
- * ``root_dir__update_if_you_intend_to_use_monorepo``: name of the root directory. The root directory name should be specified if monorepo will be used for the project. 
-   Otherwise, user can leave it blank to use the name of the current project as root directory name.
- * ``cloud``: Cloud provider you use with Databricks (AWS, Azure, or GCP)
- * ``cicd_platform`` : CI/CD platform of choice (GitHub Actions or GitHub Actions for GitHub Enterprise Servers or Azure DevOps)
+ * ``input_project_name``: name of the current project
+ * ``input_root_dir``: name of the root directory. The root directory name should be specified if monorepo will be used for the project. 
+   Otherwise, user can use the name of the current project as root directory name.
```
Comment (arpitjasa-db) [inline README.md:52]: Leaving it blank is no longer an option? We should make it clearer:
```suggestion
 * ``input_root_dir``: name of the root directory. It is recommended to use the name of the current project as the root directory name, except in the case of a monorepo with other projects where the name of the monorepo should be used instead.
```

File: README.md [hunk 2]
```diff
@@ -174,12 +173,12 @@ of the example project:
 ```
 # Note: update MLOPS_STACK_PATH to the path to your local checkout of the stack
 MLOPS_STACK_PATH=~/mlops-stack
-cookiecutter "$MLOPS_STACK_PATH" --config-file "$MLOPS_STACK_PATH/tests/example-project-configs/azure/azure-devops.yaml" --no-input --overwrite-if-exists
+databricks "$MLOPS_STACK_PATH" --config-file "$MLOPS_STACK_PATH/tests/example-project-configs/azure/azure-devops.json"
```
Comment (arpitjasa-db) [inline README.md:176]: just `databricks`? not `databricks bundle`?

**Review by arpitjasa-db — APPROVED** [review]
Looks great! Minor comments. Can we confirm there are no other instances of `cookiecutter` or its syntax in the docs?

File: README.md (hunk 1 above)
Comment (mingyu89) [reply README.md:52]: Yes. Leave it black will use the default value `my-mlops-project`. Good suggestion.

File: README.md (hunk 2 above)
Comment (mingyu89) [reply README.md:176]: Good catch. Should be `databricks bundle init`

**Comment by mabreuortega** [comment]
Having issues installing using the new databricks bundle init

**Pull request merged**